src/main/kotlin/com/jobtracker/api/security/WebSecurityConfig.kt

```diff
@@ -28,7 +28,7 @@ class WebSecurityConfig : WebSecurityConfigurerAdapter() {
   @Bean
   fun jwtDecoder(): JwtDecoder {
     val jwtDecoder = JwtDecoders.fromIssuerLocation(issuer) as NimbusJwtDecoder
-    val withIssuer = JwtValidators.createDefaultWithIssuer("accounts.google.com")
```

Comment:
I don't think this is an OK change. I'm pretty sure the https:// is _seemingly random_ from Google's jwt if you can believe it. @dakotajunkman can you please confirm?